### PR TITLE
New (facial) hair styles

### PR DESCRIPTION
**Pull request by Weisyl**
Added several new hair and facial hair styles. Credits to Fury McFlurry for the sprites.

--Hair Styles--

Birdnest: http://puu.sh/lUehj/efb4dd7df7.png

Unkept: http://puu.sh/lUek2/03c9db5f58.png

Duelist: http://puu.sh/lUifn/8d7c696ad1.png

Fastline: http://puu.sh/lUesm/1fb8233504.png

Modern: http://puu.sh/lUetO/2756f240bf.png

Quadcurls: http://puu.sh/lUevn/33ca437b69.png

Rapunzel: http://puu.sh/lUezc/c8f857acd7.png

Unshaven Mohawk: http://puu.sh/lUeBL/f51bf9935d.png

Twincurls: http://puu.sh/lUeEL/a43e206802.png

Upper Twincurls: http://puu.sh/lUeHj/0118f9f6c8.png

--Facial Hair Styles--

Britstache: http://puu.sh/lUgem/3f3d172810.png

Martial Artist: http://puu.sh/lUeOa/a0767bd256.png

Moonshiner: http://puu.sh/lUePX/d0b6ed8b8b.png

Tri-beard: http://puu.sh/lUeRs/78d781d6ff.png

Unshaven: http://puu.sh/lUeSc/4286eafe96.png

### CODE REVIEW

**Comment by Chronitonity** [comment]
FASSSSSSSSSSSSSSTLINE

**Comment by FuryMcFlurry** [comment]
Aw yiss

**Comment by danhibiki** [comment]
:ok_hand: now just add a way for lizard people to get their hair back

**Comment by FuryMcFlurry** [comment]
also in description duelist and unkept are same image.

File: code/modules/mob/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -560,6 +597,26 @@
...
 
+/datum/sprite_accessory/facial_hair/britstache
+	name = "Britstache"
+	icon_state = "facial_britstache"
+
+/datum/sprite_accessory/facial_hair/martial_artist
+	name = "Martial Artist"
+	icon_state = "facial_martialartist"
+
+/datum/sprite_accessory/facial_hair/moonshiner
```
Comment (FuryMcFlurry) [inline code/modules/mob/new_player/sprite_accessories.dm]: facial_watson , thats the wrong one m8

File: code/modules/mob/new_player/sprite_accessories.dm [hunk 2]
```diff
@@ -560,6 +597,26 @@
...
+	name = "Moonshiner"
+	icon_state = "facial_watson"
+
```
Comment (FuryMcFlurry) [inline code/modules/mob/new_player/sprite_accessories.dm:611]: icon_state = "facial_watson"

pretty sure thats wrong.

**Comment by FuryMcFlurry** [comment]
Reminder that the Modern Hairstyle isnt the sprite I gave you derp.

**Comment by Foosili** [comment]
Fuck yiss

**Pull request merged**